src/plugins/management/public/plugin.ts

```diff
@@ -120,15 +122,23 @@ export class ManagementPlugin
       euiIconType: 'logoElastic',
       category: DEFAULT_APP_CATEGORIES.management,
       updater$: this.appUpdater,
-      async mount(params: AppMountParameters) {
+      mount: async (params: AppMountParameters) => {
         const { renderApp } = await import('./application');
-        const [coreStart] = await core.getStartServices();
+        const [coreStart, deps] = await core.getStartServices();
 
         return renderApp(params, {
           sections: getSectionsServiceStartPrivate(),
           kibanaVersion,
           coreStart,
-          setBreadcrumbs: coreStart.chrome.setBreadcrumbs,
+          setBreadcrumbs: (newBreadcrumbs) => {
+            if (deps.serverless) {
+              // drop the root management breadcrumb in serverless because it comes from the navigation tree
+              const [, ...trailingBreadcrumbs] = newBreadcrumbs;
```

Comment:
This looks hacky to me as we rely on the first breadcrumb is always the root "stack management" and we drop it. An alternative could have been to pass an additional `setServerlessBreadcrumbs` function inside the `renderApp` and have if/else there. But I thought keeping this here is nicer and more contained.

Reply:
LGTM